utils/clflags.mli

```diff
@@ -272,3 +272,10 @@ val print_arguments : string -> unit
 
 (* [reset_arguments ()] clear all declared arguments *)
 val reset_arguments : unit -> unit
+
+val auto_include_otherlibs :
```

Comment:
I am not sure that I follow the reason why this function is in `Clflags`? My mental model is that `Clflags` does not interpret argument itself?

Reply:
Is there somewhere slightly higher up where it can live in ocamlcommon?

Reply:
The problem is that `Compmisc` would be an obvious place, except that `Ast_mapper` uses it...